src/sentry/notifications/models/notificationsettingoption.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
```

Comment:
Should we also have a `sentry/models/notifications/models/__init__.py` that imports all the models from notifications app? We may need the barrel file to appease django's app startup.

Reply:
oof I forgot. I'm rusty 🫠 I also need to add to INSTALLED_APPS